configuration/filters.md

```diff
@@ -39,51 +47,87 @@ Below example will grab the log lines that are error related and discard other l
     pattern: "error|ERROR|ERR|Err"
 ```
 
-Using negation it is possible to discard the specified content and process the rest.
-
-Below example will discard any debugging information from the input source to focus on more important events:
+Negative filters are also possibla via *negate* option.
+Below example will discard DEBUG logs and only pass thru other logs:
 
 ```yaml
   - name: not_debug
-    pattern: "Debug|DEBUG"
+    pattern: "  DEBUG "
```

Comment:
hmm do we need double space before Debug,.

Reply:
no we don't. I just added tabs before and after but removing to avoid confusion